apps/studio/pages/project/[ref]/auth/sessions.tsx

```diff
@@ -0,0 +1,41 @@
+import { PermissionAction } from '@supabase/shared-types/out/constants'
+
+import { SessionsAuthSettingsForm, SmtpForm, ThirdPartyAuthForm } from 'components/interfaces/Auth'
+import AuthLayout from 'components/layouts/AuthLayout/AuthLayout'
+import { ScaffoldContainer, ScaffoldHeader, ScaffoldTitle } from 'components/layouts/Scaffold'
+import NoPermission from 'components/ui/NoPermission'
+import { GenericSkeletonLoader } from 'components/ui/ShimmeringLoader'
+import { useCheckPermissions, usePermissionsLoaded } from 'hooks/misc/useCheckPermissions'
+import type { NextPageWithLayout } from 'types'
+
+const AuthSettings: NextPageWithLayout = () => {
+  const isPermissionsLoaded = usePermissionsLoaded()
+  // TODO: check if these permissions cover third party auth as well
+  const canReadAuthSettings = useCheckPermissions(PermissionAction.READ, 'custom_config_gotrue')
+
+  return (
+    <>
+      <ScaffoldContainer>
+        <ScaffoldHeader>
+          <ScaffoldTitle>Sessions</ScaffoldTitle>
+        </ScaffoldHeader>
+      </ScaffoldContainer>
+      <ScaffoldContainer className="flex flex-col gap-10" bottomPadding>
+        {!isPermissionsLoaded ? (
+          <GenericSkeletonLoader />
+        ) : !canReadAuthSettings ? (
+          <NoPermission isFullPage resourceText="access your project's authentication settings" />
+        ) : (
+          <>
+            <SessionsAuthSettingsForm />
```

Comment:
nit: existing problem on prod, but its more obvious now since we no longer need to scroll down to find this section
this callout shows up prematurely then goes away after a second (assuming that it was when the request to check the org subscription completes) haven't looked into the code yet but im perhaps can hide the callout until the subscription query is completed? (and disable the form inputs too until the subscription query is completed)

![image](https://github.com/user-attachments/assets/574ead1f-05c7-4965-86a1-895b07330731)

Reply:
out of scope i think. we are just moving stuff for this PR